chore(release): bump to v6.1.0 alpha.4 (#11461)
OpenTrons/opentrons-api: app-shell/build/release-notes.md

- Improvements to the run log, including properly listing labware that is on top of a module.
- Interface fixes and improvements.

> - Fixed a behavior where starting tip-length calibration for not-attached pipette silently failed

Reviewer: ```suggestion
- Tip Length Calibration no longer silently fails when the chosen pipette is not attached
```

Author: Nice

> - Fixed factory reset options' description

Reviewer: ```suggestion
- Clarified what data is deleted when performing a factory reset of protocol run history
```
Just a little more detail on what we added here.

Author: awesome
